lib/solvers/HighDensitySolver/MultiHeadPolyLineIntraNodeSolver/MultiHeadPolyLineIntraNodeSolver3_ViaPossibilitiesSolverIntegration.ts

```diff
@@ -48,10 +48,9 @@ export class MultiHeadPolyLineIntraNodeSolver3 extends MultiHeadPolyLineIntraNod
     viaSolver.solve()
 
     if (viaSolver.failed || !viaSolver.solved) {
-      this.failed = true
-      this.error =
-        viaSolver.error ?? "ViaPossibilitiesSolver2 failed to find a solution."
-      console.error(this.error)
+      console.error(
+        viaSolver.error ?? "ViaPossibilitiesSolver2 failed to find a solution.",
+      )
```

Comment:
We might be able to remove this logging completely, it's not really relevant to know that one iteration of the `ViaPossibilitiesSolver2` failed

Reply:
yes i agree, it goes off a lot in console as well!! We should probably have all of our logging done within a ctx.logger so that it can be enabled/disabled depending on environment